src/Text/LLVM/PP.hs

```diff
@@ -1046,12 +1043,15 @@ ppDICompileUnit' pp cu = "!DICompileUnit"
        , pure ("splitDebugInlining:"    <+> ppBool (dicuSplitDebugInlining cu))
        , pure ("debugInfoForProfiling:" <+> ppBool (dicuDebugInfoForProf cu))
        , pure ("nameTableKind:"         <+> integral (dicuNameTableKind cu))
-       , pure ("rangesBaseAddress:"     <+> ppBool (dicuRangesBaseAddress cu))
+       ]
+       ++ if llvmVer < 11 then [] else
```

Comment:
Perhaps worth adding a Haddock comment to `dicuRangesBaseAddress` ([here](https://github.com/elliottt/llvm-pretty/blob/e586747ce3f4a37245b56745b90d4c1c02423baf/src/Text/LLVM/AST.hs#L1498)) that it is only pretty-printed with LLVM 11 or later, the version that it was added.